packages/cli/src/commands/yargs-types.ts

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
that's weird, why is this not in the package typings ... I guess rhetorical question

Reply:
In most cases this types are not needed.
Like
```
cmd.command(
  ['deploy [script]'],
  '🆙 Deploy your Worker for Platforms to Cloudflare.',
  // builder
  (yargs: Argv) =>
    yargs
      .positional('script', {
        describe: 'The path to an entry point for your worker',
        type: 'string',
        requiresArg: true,
        demandOption: true,
      })
      .option('name', {
        describe: 'Name of the worker',
        type: 'boolean',
        demandOption: true,
      }),
  // handler
  async (options) => {
    console.log(options.name);
  }
);
```
will be extracted automatically.
Cloudflare made 2 things:
1. Allow  command builder to be in the other file
2. Make handler more strict (removing [key:string]: unknown) from type so `options.anyRandomName` start giving an error (in original yargs it is not an error just `unknown`)

Reply:
thats intetresting info that would be nice in there